README.md

```diff
@@ -145,6 +143,10 @@ Bootstrap = []
 [LogCfg]
 Level = "info"
 Path = "./gnfd-sp.log"
+# metrics configuration
+[MetricsCfg]
+Enabled = false
+HTTPAddress = "localhost:24036"
```

Comment:
Could you add some description for the key configurations? E.g. gateway endpoint should be equal with the endpoint on the chain.

Reply:
OK, I'll add this in next pr.